tools/es5to6.json

```diff
@@ -159,14 +159,12 @@
     "lib/client-rects.js",
     "lib/closest.js",
     "lib/config.js",
-    "lib/contains.js",
```

Comment:
looks like you've got some unmarked changes from another PR branch in this. not a problem in this case, but generally speaking it's easier to review PRs if you can hold off on ones that depend on others till they're merged